test/verify/check-networking-firewall

```diff
@@ -280,7 +313,47 @@ class TestFirewall(MachineCase):
         set_field("#tcp-ports", "80-90-100", "")
         check_error("Invalid range")
 
+    @skipImage("Filtering got fixed, and IDs changed to data-id in 184", "rhel-7-6-distropkg")
+    def testMultipleZones(self):
+        b = self.browser
+        m = self.machine
 
+        workSource = "192.168.1.0/24"
+        m.execute("firewall-cmd --permanent --zone=work --change-source={}".format(workSource))
+        m.execute("firewall-cmd --reload")
+
+        def addServiceToZone(service, zone):
+            b.click("caption button.btn-primary")
+            b.wait_present("#add-services-dialog")
+            b.click("#add-services-dialog input[value='{}']".format(zone))
+            b.click("#add-services-dialog .list-view-pf input[data-id='{}']".format(service))
+            b.click("#add-services-dialog .modal-footer .btn-primary")
+            b.wait_not_present("#cockpit_modal_dialog")
+            b.wait_present("tr[data-row-id='{}']".format(service))
+
+        self.login_and_go("/network/firewall")
+        b.wait_present("#zones-listing:contains({})".format(workSource))
+        b.wait_present("#zones-listing:contains(Work)")
+
+        addServiceToZone("pop3", "work")
+        b.wait_present("tr[data-row-id='pop3']:contains(Work)")
+        self.assertNotIn("Public", b.text("tr[data-row-id='pop3']"))
+        addServiceToZone("pop3", "public")
+        b.wait_present("tr[data-row-id='pop3']:contains(Public)")
```

Comment:
Can you please add a `firewall-cmd` CLI check that the state is really as expected?

Reply:
add this in `addServiceToZone` helper